src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+    protected String date;
+    private static final String ICON = "[E]";
+
+    public Event(String description, String date) {
+        super(description);
+        this.date = date;
+    }
+
+    @Override
+    public String printDescription() {
+
```

Comment:
Is this an unnecessary line spacing?

Reply:
Thanks for the feedback! Fixed the line spacing issue.